gcc/testsuite/gdc.dg/link.d

```diff
@@ -0,0 +1,10 @@
+// { dg-do link { target aarch64*-*-* arm*-*-* i?86-*-* x86_64-*-* } }
```

Comment:
This test is here because there is no equivalent `-main` compiler switch in gdc.  This tests the change in typeinfo.cc.